terraform/cluster/aws/main.tf

```diff
@@ -274,3 +274,27 @@ resource "null_resource" "wait_eks_addons" {
     aws_eks_addon.kube_proxy
   ]
 }
+
+resource "aws_autoscaling_schedule" "poweroff" {
```

Comment:
if I understand correctly poweroff and poweron are mutually exclusive correct?
i can either set a schedule to be off(and when it doesn't fall into that schedule will be on by default, or set an on schedule(outside schedule will be off)

Let's also specify that in the documention